doc/getting-started.rst

```diff
@@ -1,11 +1,12 @@
-Getting Started with Haskell and Cabal
+Getting Started
```

Comment:
Extreme nitpic, following line could be lenght-adjusted

Reply:
Done